Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -138,6 +138,12 @@ WI.FontStyles = class FontStyles
 
         const createIfMissing = true;
         let cssProperty = this._effectiveWritablePropertyForName(targetPropertyName, createIfMissing);
+
+        if (!targetPropertyValue.includes("!important")) {
+            targetPropertyValue += "!important";
```

Comment:
This will change *always* add the `!important` suffix to the written property value. 
It should be added only when the property value collected from the page originally was marked with `!important`.

To find identify that, you can update `WI.FontStyles._computeProperty()` to look at the `important` property of the `WI.CSSProperty` collected into `WI.DOMNodeStyles`.

Like this: 
```js
// returns true if the CSS property was authored with `!important` in its value (which exists on WI.CSSProperty._rawValue)
style.domNodeStyle.effectivePropertyForName(name).important
```

Update the object returned by `WI.FontStyles._computeProperty` with an additional property, like `isImportant` or similar.

>[!NOTE]
>The name of that return object is misleadingly pluralised now as `resultProperties`. It should be `resultProperty` since that method operates on a single property. Probably a typo from when it was written.

If you follow that return object, you'll notice it gets collected with the other ones for other CSS properties in `WI.FontStyles._propertiesMap`. 

That's the map you can then check against in `WI.FontStyles.writeFontVariation()` to check whether the value of the CSS property you're trying to update was originally marked important. If so, concatenate the "!important" to `targetPropertyValue`.

You don't have to make any changes outside of `WI.FontStyles` to fix this bug. Please don't change `WI.CSSProperty`